src/main/resources/db/changelog/changes/15-create-order-table.yaml

```diff
@@ -0,0 +1,50 @@
+databaseChangeLog:
+  - changeSet:
+      author: Serj
+      id: create-orders-table
+      changes:
+        - createTable:
+            tableName: orders
+            columns:
+              - column:
+                  name: id
+                  type: BIGINT
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+              - column:
+                  name: user_id
+                  type: BIGINT
+                  constraints:
+                    nullable: false
+              - column:
+                  name: status
+                  type: varchar(255)
```

Comment:
you even have simple varchar type here, so that would be much simpler approach instead of using converters and stuff